builtin/game/register.lua

```diff
@@ -172,6 +175,26 @@ function core.register_item(name, itemdef)
 	register_item_raw(itemdef)
 end
 
+function core.unregister_item(name)
+	if not core.registered_items[name] then
+		core.log("warning", "Not unregistering item " .. name .. " because it doesn't exist.")
```

Comment:
This could be done like with the register function: Throw error/warning and exit the function.
```
    return
end
```
It also removes a whole level of indents below.